corehq/blobs/mixin.py

```diff
@@ -31,23 +32,41 @@
 import six
 
 
-class BlobMeta(DocumentSchema):
-    id = StringProperty()
+class BlobMetaPointer(DocumentSchema):
```

Comment:
Have you considered `BlobMetaReference` as the name?

Reply:
After thinking this over more, the term "pointer" could conjure ideas about the concept of pointers in C (the programming language), which isn't quite analogous here. Renamed to `BlobMetaRef` in c50c4eed51f69366d0a71d7bb3b76a6f46151a6b